config_manager.go

```diff
@@ -353,3 +371,12 @@ func (cm *ConfigManager) ExcludeFiles() []*regexp.Regexp {
 func (cm *ConfigManager) ExcludePatterns() []*regexp.Regexp {
 	return *cm.Config.ExcludePatterns
 }
+
+func (cm *ConfigManager) printValidationWarnings() {
+	cm.Config.printValidationWarnings()
+	for _, fileToWatch := range cm.Files() {
+		if _, err := os.Stat(fileToWatch); os.IsNotExist(err) {
+			log.Warningf("Validation: file does not exist: %s", fileToWatch)
```

Comment:
Good point! Not sure how I missed that. Although, it still seems less than ideal having duplicate errors when running with `-D`. Maybe we don't need to print this when daemonized as the file could appear later on. Leave it with me and I'll think on it.